File: stara/2016/c++/main.cpp
#include "zad5_1.cpp"
#include "zad5_2.cpp"
#include "zad5_3.cpp"
#include <fstream>
using namespace std;

int main()
{
    cout << zad5_1() << endl
         << endl
         << zad5_2() << endl
         << endl
         << zad5_3();

    ofstream result("wyniki.txt");

    result << zad5_1() << endl
           << endl
           << zad5_2() << endl
           << endl
           << zad5_3();
}

File: stara/2016/python/main.py
import zad5_1
import zad5_2
import zad5_3

# zebranie odpowiedzi z podpunktow i zapisanie ich do pliku
result = '\n'.join([zad5_1.answer, zad5_2.answer,
                    zad5_3.answer])
with open('wyniki.txt', mode='w') as f:
    f.write(result)
